cmd/bbolt/main.go

```diff
@@ -1050,9 +1051,23 @@ func (cmd *benchCommand) Run(args ...string) error {
 	}
 
 	// Print results.
-	fmt.Fprintf(cmd.Stderr, "# Write\t%v(ops)\t%v\t(%v/op)\t(%v op/sec)\n", writeResults.CompletedOps(), writeResults.Duration(), writeResults.OpDuration(), writeResults.OpsPerSecond())
-	fmt.Fprintf(cmd.Stderr, "# Read\t%v(ops)\t%v\t(%v/op)\t(%v op/sec)\n", readResults.CompletedOps(), readResults.Duration(), readResults.OpDuration(), readResults.OpsPerSecond())
+	if options.GoBenchOutput {
+		// below replicates the output of testing.B benchmarks, e.g. for external tooling
+		maxLen := 14 // the length of "BenchmarkWrite"
```

Comment:
Why don't we define this as

```
const maxLen = len("BenchmarkWrite")
```

Therefore, we can drop the comment to explain where it comes from.

Reply:
sure, it's basically the re-implementation of the testing.B code that will iterate over all test names :) I'll adjust, one sec

Reply:
made it more explicit and refactored the duplicated code now, PTAL